### DISCUSSION TITLE
group_by_field not working?

### DISCUSSION THREAD

**Discussion by dsosnoski** [milvus-io/milvus#36826] [Q&A and General discussion] [answered]
I'm running milvus-lite 2.4.10 for some tests, everything seems to be working well except for group_by_field. The current documentation seems to show that the call should be something like this:
```
res = client.search(
    collection_name='knowable',
    data=query_vectors,
    limit=10,
    search_params={
        'metric_type': 'COSINE',
        'params': {'group_by_field': 'title', 'group_size': 2,},
    },
    output_fields=['title', 'text', 'category', 'paragraph_from', 'paragraph_count'],
)
```
This call executes, but the grouping parameters are ignored and I get the same results with or without them. Some older examples show using `group_by_field='title'` as direct parameters of the search call, but that gives an `invalid parameter` error now.

Can anyone tell me how to do this correctly?

**Comment by junjiejiangjjj** [comment, marked as answer]
Currently, there is a problem with the group by function of milvus-lite. It will be fixed in the next version.

**Reply by junjiejiangjjj** [reply]
https://github.com/milvus-io/milvus-lite/issues/214
